sc/src/main/java/uk/ac/ebi/atlas/experimentpage/CellMetadataService.java

```diff
@@ -96,7 +97,7 @@ public CellMetadataService(IdfParser idfParser, SolrCloudCollectionProxyFactory
         return queryResult.getOrDefault(SingleCellAnalyticsCollectionProxy.FACTORS.name(), Collections.emptyList())
                 .stream()
                 .filter(factor -> !factor.toString().equalsIgnoreCase("single_cell_identifier"))
-                .map(factor -> SingleCellAnalyticsCollectionProxy.factorAsSchemaField(factor.toString()))
+                .map(factor -> SingleCellAnalyticsCollectionProxy.factorAsSchemaField(factor.toString(), StringUtils.capitalize(factor.toString())))
                 .toArray(SingleCellAnalyticsSchemaField[]::new);
```

Comment:
It seems to me that the logic that sets the `displayName` should be encapsulated in `characteristicAsSchemaField` and `factorAsSchemaField`, so that you need to pass a single argument.

Reply:
Hmm it might seem like it from the way it's been used, but I think my logic was related to something like the species characteristic. There you wouldn't want the name of the characteristic to be the same as the display name. But seeing as it's not used like this at the moment, I will refactor it.